http_logs/index-runtime-fields.json

```diff
@@ -0,0 +1,100 @@
+{
+  "settings": {
+    "index.number_of_shards": {{ number_of_shards | default(5) }},
+    "index.number_of_replicas": {{ number_of_replicas | default(0) }},
+    "index.requests.cache.enable": false
+  },
+  "mappings": {
+    "dynamic": "strict",
+    "_source": {
+      "enabled": {{ source_enabled | default(true) | tojson }}
+    },
+    "properties": {
+      "@timestamp": {
+        "format": "strict_date_optional_time||epoch_second",
+        "type": "date"
+      },
+      "message": {
+        "type": "wildcard",
+        "fields": {
+          "keyword": {
+            "type": "keyword"
+          }
+        }
+      }
+    },
+    "runtime": {
+      {%- set sources = [('source', 'message.source'), ('wildcard', 'message'), ('keyword', 'message.keyword')] %}
+      {%- for source_type, field in sources %}
```

Comment:
The different fields sources are truly copy and paste so I did it with jinja to make this a little easier to read. I just can't read a lot of copy and paste. Well, I can. But I can't tell if there are subtle differences.